server/accounts.go

```diff
@@ -793,6 +817,23 @@ func (a *Account) checkExpiration(claims *jwt.ClaimsData) {
 	a.expired = false
 }
 
+// hasIssuer returns true if the issuer matches the account
+// issuer or it is a signing key for the account.
+func (a *Account) hasIssuer(issuer string) bool {
+	a.mu.RLock()
+	defer a.mu.RUnlock()
+	// same issuer
+	if a.Issuer == issuer {
+		return true
+	}
+	for _, v := range a.signingKeys {
```

Comment:
Would use a `for i:=0;i<len(..` here to avoid copies.

Reply:
fixed